src/serialize.h

```diff
@@ -898,16 +978,16 @@ void SerializeMany(Stream& s)
 }
 
 template<typename Stream, typename Arg>
-void SerializeMany(Stream& s, Arg&& arg)
+void SerializeMany(Stream& s, const Arg& arg)
 {
-    ::Serialize(s, std::forward<Arg>(arg));
+    ::Serialize(s, arg);
```

Comment:
The reason for removing the `std::forward` calls here is explained in the commit message (there is no benefit in passing down the rvalue-ness).